app/views/posts/show.html.erb

```diff
@@ -4,7 +4,10 @@
     <p class="post-likes-comments">Comments: <%= @post.comment_counter %>, Likes: <%= @post.like_counter %></p>
     <p><%= @post.text %></p>
             <div class="like-comment-btn">
-      <%= form_with model: @like, url: user_post_likes_path(user_id: @post.author.id, post_id: @post.id), local: true do |form| %>
+      <%# <%= form_with model: @like, url: user_post_likes_path(user_id: @post.author.id, post_id: @post.id), local: true do |form| %>
+        
```

Comment:
I will leave the comment here since you do not have any changes on the posts/index.html.erb file.

### Clickable post
[OPTIONAL] It would be best if you made the post title clickable so that users can be able to navigate to the post show page.

![image](https://user-images.githubusercontent.com/38088818/232016056-3bae058f-d45d-4460-ac3e-a35db84ec310.png)